data/tooltips.txt

```diff
@@ -329,8 +329,8 @@ tip "radar jamming:"
 tip "ramscoop:"
 	`Replenishes fuel by harvesting the stellar wind. Ramscoops are more effective when closer to a star. If you add more than one ramscoop, each additional one is less effective: you need four ramscoops to achieve double the recharge rate that one ramscoop provides.`
 
-tip "required crew / bunks:"
-	`How many crew members are required to operate this ship, and how many total bunks are available. Extra bunks can be used for passengers or extra crew.`
+tip "bunks free:"
```

Comment:
Not too keen on this change. While it does flip the order to display the number of bunks free, which is more useful for running jobs than the number of required crew, this now doesn't communicate why some bunks are taken up. I think we should just leave this line how it was, displaying the number of required crew and the total number of bunks.